<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionAnimationTab</class>
 <widget class="QWidget" name="OptionAnimationTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>105</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>304</width>
    <height>105</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox_bgColor">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_bgColor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>BG Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imagePath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>画像パス</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_useImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>背景画像を使う</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_imagePath">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_imagePath">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
